charts/portal/README.md

```diff
@@ -461,6 +461,9 @@ this feature.
 | `rabbitmq.serviceAccount.create`| Enable creation of ServiceAccount for Bitnami RabbitMQ |`true`|
 | `rabbitmq.serviceAccount.name`| Name of the created serviceAccount | Generated using the `portal.fullname` template |
 | `rabbitmq.rbac.create`| Create & use RBAC resources |`true`|
+| `rabbitmq.volumePermissions.enabled` | Enable init container that changes the owner and group of the persistent volume(s) mountpoint to `runAsUser:fsGroup`. After enabling set the appropriate runAsUser and fsGroup values |`false`|
+| `rabbitmq.podSecurityContext.fsGroup` | Set RabbitMQ pod's Security Context fsGroup | `1001` |
```

Comment:
Do we have to set podSecurityContext.enabled to true if we have to pass custom value for rabbitmq.podSecurityContext.fsGroup?

Reply:
podSecurityContext .enabled is true by default